### DISCUSSION TITLE
Games run too fast

### DISCUSSION THREAD

**Discussion by PunchyMcHurtyFist** [visualboyadvance-m/visualboyadvance-m#1226] [General]
On a fresh install of VBA-M 64bit, latest release 2.0.1 on Windows 8.1, games run way too fast. Whenever I use the speed up button, it actually makes the game run as normal speed, whenever I release it, it speeds up again.

**Comment by ZachBacon** [comment]
Adjust your throttle.

**Comment by ChaosTFWater** [comment]
How do I adjust the throttle.

**Comment by rkitover** [comment]
Options -> General

**Comment by rkitover** [comment]
@ericjwg this is fixed in the nightlies

**Comment by ghost** [comment]
Try to enable V-Sync
